src/plugins/condition/plugin.js

```diff
@@ -0,0 +1,39 @@
+/*****************************************************************************
+ * Open MCT, Copyright (c) 2014-2018, United States Government
```

Comment:
Change copyright to 2014-2019

Reply:
BTW - I found 788 hits on the 2018 copyright. Changed it just for the files I'm adding for now. We'll need to do it again in two weeks.